code/game/turfs/turf.dm

```diff
@@ -543,10 +572,46 @@ GLOBAL_LIST_EMPTY(station_turfs)
 			if(nutri_check.nutriment_factor >0)
 				M.reagents.remove_reagent(R.type, min(R.volume, 10))
 
-//Whatever happens after high temperature fire dies out or thermite reaction works.
-//Should return new turf
+///Prot to set the damage of the turf, also used to call the windows and doors adjacent_fire_act()
+/turf/proc/set_heat_damage_amount(exposed_temperature)
+	var/obj/structure/window/Window
+	for(Window in range(1, src))
```

Comment:
What's with this format, why.

Reply:
wym? should i just for(var/obj/structure/window/Window in range(1, src)) ?

Reply:
Yep, any reason not to?

Reply:
not really, i'll change it

Reply:
didnt even know this worked